index.js

```diff
@@ -59,7 +63,9 @@ module.exports = app => {
           tag_name: releaseInfo.tag,
           body: releaseInfo.body,
           draft: true,
-          prerelease: config.prerelease
+          prerelease: config.prerelease,
+          target_commitish:
+            config['target-branch'] || context.payload.repository.default_branch

```

Comment:
the default should be handled by the schema.
```suggestion
            config['target-branch']
```